### PR TITLE
Sdk 518 improve code example node

### PR DESCRIPTION
**Pull request by shalevr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,10 @@ const userOptions = {
 };
 
 await ciscoTracing.init(userOptions);
+
+// The instrument requires should be called after ciscoTracing init
```
Comment (haddasbronfman) [inline README.md:63]: I suggest: The import of you instrumented library should go her, **after** the ciscoTracing.init()

File: README.md [hunk 2]
```diff
@@ -71,6 +75,10 @@ const userOptions: Partial<Options> = {
   ciscoToken: 'cisco-token',
 };
 await ciscoTracing.init(userOptions);
+
+// The instrument imports should be called after ciscoTracing init
```
Comment (haddasbronfman) [inline README.md:79]: same

File: README.md (hunk 1 above)
Comment (shalevr) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (shalevr) [reply README.md]: Done

File: README.md [hunk 3]
```diff
@@ -71,6 +75,10 @@ const userOptions: Partial<Options> = {
   ciscoToken: 'cisco-token',
 };
 await ciscoTracing.init(userOptions);
+
+// The import of your instrumented library should go her, after the ciscoTracing.init()
```
Comment (arbiv) [inline README.md:79]: here

**Review by arbiv — CHANGES_REQUESTED** [review]

**Pull request closed without merging**